src/logic/redux/store.js

```diff
@@ -0,0 +1,25 @@
+import { createLogger } from 'redux-logger';
+import { persistStore } from 'redux-persist';
+import createSagaMiddleware from 'redux-saga';
+import { legacy_createStore, applyMiddleware } from 'redux';
+
+import sagas from '../sagas';
+import rootReducer from './rootReducer';
+import apiMiddleware from './apiMiddleware';
+
+const loggerMiddleware = createLogger({
+  collapsed: (_getState, _action, logEntry) => !logEntry.error,
+  diff: true,
+});
+const sagaMiddleware = createSagaMiddleware();
+
+let middlewares = [apiMiddleware, sagaMiddleware];
+middlewares.push(loggerMiddleware);
+
+const store = legacy_createStore(rootReducer, applyMiddleware(...middlewares));
+const persistor = persistStore(store, {}, () => {});
```

Comment:
Can you tell me what are the arguments in this case?

Reply:
store, persistorConfig and callback